### ISSUE TITLE
Lint incorrectly errors on quoted bool

### ISSUE THREAD

**Issue by elmobp** [rodjek/puppet-lint#844]
If you have a look at this page https://puppet.com/docs/puppet/5.3/lang_data_abstract.html#variant

You will see it says to use the Variant as such
```puppet
Variant[Enum['true', 'false'], Boolean]
```

If you specify it like that puppet-lint throws an error 
WARNING: quoted boolean value found on line 4

Should it be able to skip as its a parameter? Rather then inside the actual manifest
